CHANGELOG.rst

```diff
@@ -1,3 +1,9 @@
+**0.1.0 - 03/31/23**
+
+ - Implemented W2/1099 forms
+ - Implemented typographic noise function
+ - Implemented incorrect selection noise function
```

Comment:
Do we need to address bugfixes here (eg the zipcode and age fixes)?

Reply:
Those are changes in Synth Pop

Reply:
heh, right